src/inference.jl

```diff
@@ -1986,3 +1408,397 @@ function rxinference(;
+- `datastream`: A stream of `NamedTuple` with data, required (or `data`)
+- `autoupdates = nothing`: auto-updates specification, required for streamline inference, see `@autoupdates`
+- `initmarginals = nothing`: `NamedTuple` or `Dict` with initial marginals, optional
+- `initmessages = nothing`: `NamedTuple` or `Dict` with initial messages, optional
+- `constraints = nothing`: constraints specification object, optional, see `@constraints`
+- `meta  = nothing`: meta specification object, optional, may be required for some models, see `@meta`
+- `options = nothing`: model creation options, optional, see `ModelInferenceOptions`
+- `returnvars = nothing`: return structure info, optional, defaults to return everything at each iteration, see below for more information
+- `predictvars = nothing`: return structure info, optional, see below for more information (exclusive for batch inference)
+- `historyvars = nothing`: history structure info, optional, defaults to no history, see below for more information (exclusive for streamline inference)
+- `keephistory = nothing`: history buffer size, defaults to empty buffer, see below for more information (exclusive for streamline inference)
+- `iterations = nothing`: number of iterations, optional, defaults to `nothing`, the inference engine does not distinguish between variational message passing or Loopy belief propagation or expectation propagation iterations, see below for more information
+- `free_energy = false`: compute the Bethe free energy, optional, defaults to false. Can be passed a floating point type, e.g. `Float64`, for better efficiency, but disables automatic differentiation packages, such as ForwardDiff.jl
+- `free_energy_diagnostics = BetheFreeEnergyDefaultChecks`: free energy diagnostic checks, optional, by default checks for possible `NaN`s and `Inf`s. `nothing` disables all checks.
+- `showprogress = false`: show progress module, optional, defaults to false (exclusive for batch inference)
+- `catch_exception`  specifies whether exceptions during the inference procedure should be caught, optional, defaults to false (exclusive for batch inference)
+- `callbacks = nothing`: inference cycle callbacks, optional, see below for more info
+- `addons = nothing`: inject and send extra computation information along messages, see below for more info
+- `postprocess = DefaultPostprocess()`: inference results postprocessing step, optional, see below for more info
+- `events = nothing`: inference cycle events, optional, see below for more info (exclusive for streamline inference)
+- `uselock = false`: specifies either to use the lock structure for the inference or not, if set to true uses `Base.Threads.SpinLock`. Accepts custom `AbstractLock`. (exclusive for streamline inference)
+- `autostart = true`: specifies whether to call `RxInfer.start` on the created engine automatically or not (exclusive for streamline inference) (exclusive for streamline inference)
+- `warn = true`: enables/disables warnings
```

Comment:
`catch_exception` argument explanation is missing (did we have it?)

Reply:
I believe the short explanation was given in line 1493, but the extended description was missing indeed. Added now.